parser/unparser.go

```diff
@@ -444,3 +471,76 @@ func bytesToOctets(byteVal []byte) string {
 	}
 	return b.String()
 }
+
+// Takes in a function then inserts a newline based on the rules provided in formatting options
```

Comment:
Even though this is a private function, use the same format for exported functions, e.g. `wrapForOperator inserts a newline after operators configured in the formatting options.`